app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def calculate_team_total_rating(team: list):
+    return sum([player.get_rating() for player in team])
```

Comment:
You can prevent creating a redundant list via passing comprehension without square brackets